src/eme.js

```diff
@@ -82,10 +82,14 @@ export const getSupportedKeySystem = (keySystems) => {
 
     if (!promise) {
       promise =
-        window.navigator.requestMediaKeySystemAccess(keySystem, supportedConfigurations);
+        window.navigator.requestMediaKeySystemAccess(keySystem, supportedConfigurations).catch((error) => {
+          emeError(error, videojs.Error.EMEFailedToRequestMediaKeySystemAccess);
+        });
     } else {
-      promise = promise.catch((e) =>
-        window.navigator.requestMediaKeySystemAccess(keySystem, supportedConfigurations));
+      promise = promise.catch(() => {
+        // TODO: Not sure if we want to call emeError here or not.
+        window.navigator.requestMediaKeySystemAccess(keySystem, supportedConfigurations);
+      });
```

Comment:
Here is just an accumulation of the promise,
I guess you should trigger error once in the other place

![CleanShot 2024-03-12 at 13 12 11@2x](https://github.com/videojs/videojs-contrib-eme/assets/98566601/96c8596c-c23b-4479-9d8e-6f3367bbce03)